example_workspace/src/move_base_app/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(move_base_app)
+
+find_package(catkin REQUIRED rosjava_build_tools)
+
+# Set the gradle targets you want catkin's make to run by default, e.g.
+# catkin_rosjava_setup(installDist)
+# catkin_android_setup(assembleDebug)
```

Comment:
Hmm don't we need this?

Reply:
Yes, it was commented out because move_base example had problems to build before.
Now it is corrected.